source/_integrations/google_photos.markdown

```diff
@@ -0,0 +1,81 @@
+---
+title: Google Photos
+description: Instructions on how to use Google Photos in Home Assistant.
+ha_category:
+  - Media source
+ha_iot_class: Cloud Polling
+ha_release: 2024.10
+ha_config_flow: true
+ha_domain: google_photos
+ha_codeowners:
+  - '@allenporter'
+ha_integration_type: integration
+ha_platforms:
+  - media_source
+related:
+  - docs: /integrations/media_source
+    title: Media source integration documentation
+  - url: https://photos.google.com/
+    title: Google Photos
+  - url: https://console.cloud.google.com/apis/library/photoslibrary.googleapis.com
+    title: Google Developers Console
+---
+
+The **Google Photos** integration allows you to connect your [Google Photos](https://photos.google.com/) to Home Assistant. The integration adds a [media source](/integrations/media_source) with
+recent  photos allowing you to view or cast your photos from the media browser.
+
+## Prerequisites
+
+You need to configure developer credentials to allow Home Assistant to access your Google Account.
+These credentials are the same as the ones for [Nest](/integrations/nest), [Google Tasks](/integrations/google_tasks) and [Google Mail](/integrations/google_mail).
+These are not the same as *Device Auth* credentials previously recommended for [Google Calendar](/integrations/google).
+
+If you have already set up the correct credentials, you can do step 1 and then skip to step 13 on the below instructions.
+
+{% details "Generate Client ID and Client Secret" %}
+
+This section explains how to generate a Client ID and Client Secret on
+[Google Developers Console](https://console.cloud.google.com/apis/library/photoslibrary.googleapis.com).
+
```

Comment:
could these steps be an include and be reused? I'm not asking you to do it - I can make the include. I'm asking because I'm wondering whether you actively decided against reuse for some reason.

Reply:
I copied these steps verbatim from google tasks. My impression is most of the google integrations are similar (tasks, google mail, photos,  -- and calendar now. not sure about youtube) and nest is different.

Reply:
ok, so you wouldn't mind if I moved those into an include. I guess it's outside the scope of this PR. I can do it after it's merged.

Reply:
Yeah I am supportive. Happy to review and double check everything is the same.

Reply:
thank you!